src/test/java/com/puppycrawl/tools/checkstyle/checks/AvoidEscapedUnicodeCharactersCheckTest.java

```diff
@@ -32,6 +34,109 @@
 import com.puppycrawl.tools.checkstyle.api.TokenTypes;
 
 public class AvoidEscapedUnicodeCharactersCheckTest extends BaseCheckTestSupport {
+
+    private static final int[] C0_CONTROL_CHARACTER_INDICES = {
```

Comment:
please make comment above each such  array to define from where values are taken.
Like: // C0 (ASCII and derivatives) - https://en.wiktionary.org/wiki/Appendix:Control_characters#C0_.28ASCII_and_derivatives.29

please do for that for all other arrays.